cylc/rose/stem.py

```diff
@@ -60,6 +56,29 @@
     SOURCE_<project>_REV
         The revision of the project specified on the command line. This
         is intended to specify the revision of `fcm-make` config files.
+
+USAGE EXAMPLES
+    rose stem --group=developer
+        - Installs files and workflow found in PWD.
+        - Sets variable RUN_NAMES=['developer']. This is usually used to
+          enable a set of tasks or tests.
+
+    rose stem --source=/path/to/source --source=/other/source --group=mygroup
+        - Installs:
+            - files and workflow found in /path/to/source
+            - files found at /other/source
+        - set RUN_NAMES=['mygroup']
+
+    rose stem --source=foo=/path/to/source --source=bar=fcm:bar_tr@head
+        - Installs:
+            - files and workflow found in /path/to/source
+            - files from FCM repository bar_tr at head.
+
+    rose stem --source=A=
+        - Installs files and workflow found in PWD.
+        - Sets project name = A, which will change the jinja2 variables
+          set by rose stem.
```

Comment:
Use same format as other commands.

```suggestion
Examples
    $ rose stem --group=developer
    # - Installs files and workflow found in PWD.
    # - Sets variable RUN_NAMES=['developer']. This is usually used to
    #   enable a set of tasks or tests.

    $ rose stem --source=/path/to/source --source=/other/source --group=mygroup
    # - Installs:
    #   - files and workflow found in /path/to/source
    #   - files found at /other/source
    # - set RUN_NAMES=['mygroup']

    $ rose stem --source=foo=/path/to/source --source=bar=fcm:bar_tr@head
    # - Installs:
    #   - files and workflow found in /path/to/source
    #   - files from FCM repository bar_tr at head.

    $ rose stem --source=A=
    # - Installs files and workflow found in PWD.
    # - Sets project name = A, which will change the jinja2 variables
    #    set by rose stem.
```

Reply:
I'm not sure the detailed descriptions help that much (and they aren't entirely accurate).
Perhaps just give the example commands (as in Rose 2019) and rely on the rest of to help to explain what they do?

Reply:
>  just give the example commands (as in Rose 2019) and rely on the rest of to help to explain what they do?

I don't like this approach, but it's probably good enough for now.